core/domain/role_domain.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
```

Comment:
I don't see a file called role_services_tests (which should, btw, be called role_services_test). That may be a cause.

Also, examine your import chain. Are there any circular imports? That sort of thing can cause this issue, too.